.github/.sqlfluff

```diff
@@ -0,0 +1,17 @@
+[sqlfluff]
+dialect = sqlite
+max_line_length = 180
```

Comment:
Can we get this down to 80, or at most 120?

Reply:
I've left it at this because the changelog namespaces get rendered in the most ugliest way possible. I have my limits!